dashboard/fixtures/07d_rawchem.yaml

```
@@ -8566,4 +10286,6 @@
     temp_id: 1
     temp_obj_name: ExtractedChemical
     rid: DTXRID00299
+    created_at: 2019-04-08 07:01:07.897509
+    created_by_id: 1
     dsstox: 12
```

The audit log test failure is due to RawChem 856 not having an updated_at being set.  This prevents the audit log from showing up.  This should resolve the last failing test.
```suggestion
    temp_id: 1
    temp_obj_name: ExtractedChemical
    rid: DTXRID00299
    created_at: 2019-04-08 07:01:07.897509
    updated_at: 2019-04-08 07:01:07.897509
    created_by_id: 1
    updated_by_id: 1
    dsstox: 12
```